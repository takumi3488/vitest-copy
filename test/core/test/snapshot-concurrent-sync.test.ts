import { describe, it } from 'vitest'

// from https://github.com/vitest-dev/vitest/issues/3361
describe.concurrent('concurrent suite', () => {
  it('snapshot', ({ expect }) => {
    expect({ foo: 'bar' }).toMatchSnapshot()
  })

  it('empty test')
})
