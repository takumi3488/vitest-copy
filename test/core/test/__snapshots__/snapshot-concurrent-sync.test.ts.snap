// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`concurrent suite > snapshot 1`] = `
{
  "foo": "bar",
}
`;
